CONTRIBUTING.md

```diff
@@ -0,0 +1,17 @@
+## Technical Steering Committee (TSC)
+
+The Technical Steering Committee (the “TSC”) is responsible for all technical oversight of the OSS Review Toolkit project. 
+
+TSC members:
+
+* Frank Viernau ([@fviernau](https://github.com/fviernau))
+* Martin Nonnenmacher ([@mnonnenmacher](https://github.com/mnonnenmacher))
+* Sebastian Schuberth ([@sschuberth](https://github.com/sschuberth))
+* Thomas Steenbergen ([@tsteenbe](https://github.com/tsteenbe)) 
+
+TSC has elected not to have a TSC chair, but Thomas Steenbergen as spokesperson / project manager.
+
+Voting members on [Automating Compliance Tooling](https://automatecompliance.org/):
```

Comment:
"... on the [Automating Compliance Tooling](https://automatecompliance.org/) project fund:"

Reply:
@tsteenbe: You didn't add "project fund", the sentence sounds a bit odd without it.